packages/terra-show-hide/src/_ShowHideButton.jsx

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './_ShowHideButton.module.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
```

Comment:
Can't remember if we've talked about standardizing these codes somewhere common across our components, or using a 3rd party library, if it's worth it, etc..

Reply:
I'm fine leaving as is for now. At some point, we'll want to package this up into a utility library.